Cargo.lock

```diff
@@ -4677,6 +4701,26 @@ version = "0.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "19b17cddbe7ec3f8bc800887bab5e717348c95ea2ca0b1bf0837fb964dc67099"
 
+[[package]]
+name = "pem"
+version = "0.8.2"
+source = "git+https://github.com/mesalock-linux/pem-rs-sgx#fdfef4f24a9fb3fa72e8a71bb28bd8ff15feff2f"
+dependencies = [
+ "base64 0.13.0 (git+https://github.com/mesalock-linux/rust-base64-sgx)",
+ "once_cell 1.4.0",
+ "regex 1.3.1",
+ "sgx_tstd",
+]
+
+[[package]]
+name = "pem"
+version = "1.0.2"
```

Comment:
The pem crate above is imported in the version `0.8.2` for `sgx`. Use the same version for `std` (the issue might originate in our port of the `rcgen` crate?).

Reply:
See comment above